game_event/models.py

```diff
@@ -0,0 +1,44 @@
+from django.db import models
+from game_event_player.models import GameEventPlayer
+from court.models import Court
+
+
+class BallGame(models.TextChoices):
+    Soccer = 'Soccer', 'Soccer'
+    Basketball = 'Basketball', 'Basketball'
+    Volleyball = 'Volleyball', 'Volleyball'
+    Baseball = 'Baseball', 'Baseball'
+    Tennis = 'Tennis', 'Tennis'
+    Rugby = 'Rugby', 'Rugby'
+    Golf = 'Golf', 'Golf'
+    Cricket = 'Cricket', 'Cricket'
+    Handball = 'Handball', 'Handball'
+
+
+class Rating(models.IntegerChoices):
+    One = 1
+    Two = 2
+    Three = 3
+    Four = 4
+    Five = 5
+    Six = 6
+    Seven = 7
+    Eight = 8
+    Nine = 9
+    Ten = 10
+
+
+class GameEvent(models.Model):
+    id = models.IntegerField(primary_key=True)
+    time = models.DateTimeField(null=False)
+    level_of_game = models.IntegerField(choices=Rating.choices, blank=True)
+    min_number_of_players = models.PositiveIntegerField()
+    max_number_of_players = models.PositiveIntegerField()
+    ball_responsible_player = models.ForeignKey(GameEventPlayer, on_delete=models.CASCADE)
```

Comment:
I think this part is going to be tricky. You have some sort of circular dependency here. 

You need the event to create the event player, and you need the event player to be the ball responsible. There's a way to work around this. It's mentioned here (with oddly similar case): https://stackoverflow.com/questions/8466726/django-circular-model-reference

If you don't get it figured out easily enough, I suggest having this reference a player rather than an event player for now, and open an issue to figure it out later.

Reply:
> I think this part is going to be tricky. You have some sort of circular dependency here.
> 
> You need the event to create the event player, and you need the event player to be the ball responsible. There's a way to work around this. It's mentioned here (with oddly similar case): https://stackoverflow.com/questions/8466726/django-circular-model-reference
> 
> If you don't get it figured out easily enough, I suggest having this reference a player rather than an event player for now, and open an issue to figure it out later.

Thank you for this comment, we will see what we do about it and let you know

Reply:
I have another idea for avoiding this circular dependency. We can have a ball_responsible boolean on the EventPlayer model. It will be False by default, and True for the ones who are ball responsible.

This way, we only have one way dependency.

Reply:
> I have another idea for avoiding this circular dependency. We remove this field here, and add a ball_responsible boolean on the EventPlayer model. It will be False by default, and True for the ones who are ball responsible.
> 
> This way, we only have one way dependency.

So, just to make sure I understand: on GameEvent we don't need the ball responsible, the EventPlayer entity
will be responsible about this field?

Reply:
and you can add a validation that the player with the ball is one from the players list